packages/medusa/src/api/routes/admin/tax-rates/create-tax-rate.ts

```diff
@@ -39,6 +39,9 @@ import { validator } from "../../../../utils/validator"
  *     application/json:
  *       schema:
  *         $ref: "#/components/schemas/AdminPostTaxRatesReq"
+ * x-codegen:
+ *   method: create
+ *   queryParams: AdminPostTaxRatesParams
```

Comment:
Client was not referencing the correct queryParams type.
https://github.com/medusajs/medusa/blob/d18ba712f2069818e2cb3172253109c195a33370/packages/medusa-js/src/resources/admin/tax-rates.ts#L53-L57